js/pages/estimation/cohort-comparison-manager.js

```diff
@@ -48,10 +48,10 @@ define([
 	) {
 		class CohortComparisonManager extends AutoBind(Component) {
 			constructor(params) {
-				super();
-				this.cohortComparisonId = params.currentCohortComparisonId;
-				this.cohortComparison = params.currentCohortComparison;
-				this.cohortComparisonDirtyFlag = params.dirtyFlag;
+				super(params);
```

Comment:
Please see comment in https://github.com/OHDSI/Atlas/issues/910 for how we should handle routerParams as a direct observable, and not extract each param from the routerParams() object.